pyads/ads.py

```diff
@@ -1,1153 +1,1153 @@
-"""Pythonic ADS functions.
-
-:author: Stefan Lehmann <stlm@posteo.de>

```

Comment:
You are replacing the whole file ads.py file. Please make sure only your changes for ads.py are in the PR.